src/map/clif.cpp

```diff
@@ -6599,20 +6593,23 @@ void clif_efst_status_change_sub(struct block_list *tbl, struct block_list *bl,
 			case SC_SPHERE_4:
 			case SC_SPHERE_5:
 				if( spheres_sent ){
-					target = AREA_WOS;
+					target = AREA_WOS;	// area without self
+					// check me : target should stay AREA_WOS ?
 				}
 				break;
 			case SC_HELLS_PLANT:
-				if( sc && sc->getSCE(type) ){
-					tick = sc->getSCE(type)->val4;
-				}
+				tick = sce->val4;
 				break;
 		}
 
+		int dval1 = scdb->flag[SCF_SENDVAL1] ? sce->val1 : 0;
```

Comment:
Are the val# not 0 by default? Could have sworn it was but maybe I'm blowing smoke!